### ISSUE TITLE
performance impact of AST grid tolerance to wide field images 

### ISSUE THREAD

**Issue by kswang1029** [CARTAvis/carta-frontend#451]
Currently, the minimum of  grid tolerance is set to 0.1%.
![Screen Shot 2019-08-07 at 9 48 50 AM](https://user-images.githubusercontent.com/20819712/62588537-9bfd9c00-b8f8-11e9-913c-77f1884cff17.png)
This is good for small field images in general. 

However, if it is a wide field image such as the following,
![Screen Shot 2019-08-07 at 9 51 04 AM](https://user-images.githubusercontent.com/20819712/62588599-e7b04580-b8f8-11e9-823f-9da123e924e1.png)
and if we click spinner "down" to have the tolerance to be 0.1%, the entire page becomes frozen. 1% still works fine in this case.

**Comment by veggiesaurus** [comment]
relabelled as a bug, since it can crash the frontend

**Comment by Pietersielie** [comment]
Merged by #460?

**Comment by kswang1029** [comment]
> Merged by #460?

Nope. The underlying issue remains (performance issue from astlib). #460 is just a workaround to lower the chance of having performance issue. A possible "solution" would be setting the tolerance dynamically based on the rendering area on screen.
